common.gypi

```diff
@@ -15,7 +15,7 @@
       }
     }]
   ],
-  'include_dirs': ["<!@(node -p \"require('../').include\")"],
+  'include_dirs': ["<!(node -p \"require('../').include_dir\")"],
```

Comment:
@KevinEady I think you need to rebase and drop this change to make the CI work:

| Version | Job | Status |
|---:|---|:---:|
| v15.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2948/ | ✗ |
| v14.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2949/ | ✗ |
| v12.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2950/ | ✗ |
| v10.x | https://ci.nodejs.org/job/node-test-node-addon-api-new/2951/ | ✗ |

Reply:
... because https://github.com/nodejs/node-addon-api/commit/f27623ff611645cd7d927f0de81158a93e960542 (https://github.com/nodejs/node-addon-api/pull/766) landed on the main branch.